docs/configuration/firewall/general-legacy.rst

```diff
@@ -1,10 +1,10 @@
 :lastproofread: 2021-06-29
 
-.. _firewall-legacy:
```

Comment:
if you change the reference name, you must check the usage. For example it is use [here](https://github.com/vyos/vyos-documentation/blob/a7f6b67f7e8c2d6aae9db17832ed26abbb29beee/docs/quick-start.rst?plain=1#L134)

Reply:
Excluding those issues, happy with the overall style and layout? If so I will start working on fixing it all up properly (including the references).